exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillConnectionImpl.java

```diff
@@ -760,6 +765,24 @@ public void abort(Executor executor) throws SQLException {
     }
   }
 
+  @Override
+  public boolean useAnsiQuotedIdentifiers() throws SQLException {
+    boolean systemOption = false;
+    Boolean sessionOption = null;
+    String sql = String.format("select type, bool_val from sys.options where name = '%s'",
+        PlannerSettings.ANSI_QUOTES_KEY);
+    ResultSet rs = executeSql(sql);
```

Comment:
Done.
New RPC request is implemented. RPC response returns `ServerProperties` that represent the list of server session options.